README.md

```diff
@@ -113,3 +113,4 @@ FRAMEWORK | [NAME](LINK_TO_REPO) | DESCRIPTION | `cargo shuttle init --from USER
 --------------|-----------------|-------------|----------------
 Axum/Any | [Fullstack Rust](https://github.com/TylerBloom/shuttle-fullstack-rust-example) | A basic project template for fullstack Rust projects | `cargo shuttle init --from TylerBloom/shuttle-fullstack-rust-example`
 Axum/Yew | [Web App with Yew](https://github.com/sentinel1909/shuttle-template-yew) | A basic project template for a web app using the Yew framework | `cargo shuttle init --from sentinel1909/shuttle-template-yew` |
+Actix Web | [Web API with Actix Web](A starter template for an API using the Actix Web framework) | `cargp shuttle init --from sentinel1909/shuttle-template-actix`
```

Comment:
Hey, the link to the repo is missing :) Also a little change needed: `cargp` -> `cargo`